static-site/.eslintrc.yaml

```
@@ -30,6 +30,7 @@ rules:
   react/prop-types: off # Remove this override once all props have been typed using PropTypes or TypeScript.
   '@next/next/no-img-element': off # Remove this if we use next.js optimisations for <img>
   '@next/next/no-html-link-for-pages': off
+  '@typescript-eslint/no-var-requires': off # require() is used for images that are not under static-site/public
```

doing something like this (note: did not check syntax carefully...) will make this specific to JSX/TSX files, which feels like a better scoping.


```suggestion
  overrides:
    # require() is used for images that are not under static-site/public
    - files:
      - "**.[jt]sx"
      rules:
        "@typescript-eslint/no-var-requires": off
```

---

Thanks, good suggestion! Added in [force-push](https://github.com/nextstrain/nextstrain.org/compare/d6baa4f8957c59cbbf4786d71ce4298968054983..0e566de918aae41b399800dda8e343047621d174).

Your syntax worked! I just changed indentation and quoting for consistency.